airbyte-cdk/python/unit_tests/sources/declarative/requesters/paginators/test_default_paginator.py

```diff
@@ -206,6 +213,7 @@ def test_initial_token_with_offset_pagination():
     paginator = DefaultPaginator(
         strategy, config, url_base, parameters={}, page_size_option=page_size_request_option, page_token_option=page_token_request_option
     )
+    paginator.reset()
```

Comment:
It feels like it shouldn't change anything to the test result so I'm wondering why do we reset here?

Reply:
true. removed